### ISSUE TITLE
Unable to build on Mac OS X

### ISSUE THREAD

**Issue by danielmatz** [fish-shell/fish-shell#1400]
I'm on Mac OS X 10.9.2, and I use MacPorts.  I've been unable to build since 1177daecded5b8dac03097801d4d6111330e1cff.  It looks like that change stopped looking into the MacPorts install directories.  The funny thing is, `configure` doesn't report anything is wrong.  It's not until I `make` that I get:

```
Undefined symbols for architecture x86_64:
  "_libintl_setlocale", referenced from:
      get_description(char const*) in mimedb.o
      _main in mimedb.o
ld: symbol(s) not found for architecture x86_64
collect2: error: ld returned 1 exit status
make: *** [mimedb] Error 1
```

**Comment by mattjj** [comment]
This issue came up for me after upgrading to Yosemite and Geoff's suggestion to configure with `--without-gettext` fixed it. I'm not sure if there are any downsides to doing that. (The /opt/local prefix is for macports stuff, and I'm building with gcc from macports.)

Thanks!

**Comment by ridiculousfish** [comment]
--without-gettext disables translations.

**Comment by faho** [comment]
Since the error occurs in mimedb stuff, this might be solved by #2229.

**Comment by zanchey** [comment]
Should be fixed with merge of #2235 by removing the problematic component entirely. If you can still reproduce, please speak up!
